src/main/java/com/es/phoneshop/web/ProductPriceHistoryPageServlet.java

```diff
@@ -0,0 +1,28 @@
+package com.es.phoneshop.web;
+
+import com.es.phoneshop.model.product.ArrayListProductDao;
+import com.es.phoneshop.model.product.ProductDao;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+public class ProductPriceHistoryPageServlet extends HttpServlet {
```

Comment:
Please add tests to verify that query parameters parsed correctly. You can use `ArgumentCaptor` for that